### PR TITLE
feat: add Bluesky social icon and update footer navigation

### LINKED ISSUE #7207: Add blue sky in footer link
(very) recently we joined blue sky. We'll have to put the link somewhere to let people know it exists. For example in the footer and why not a badge on the home page.

REF: https://github.com/nodejs/admin/issues/932

TODO:

- Add icon component 
- Add link to `navigation.json`

### PR DESCRIPTION
**Pull request by TenzDelek**
## Description
added Bluesky logo in the Footer

## Validation
image

## Related Issues
Fixes #7207 

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npm run format` to ensure the code follows the style guide.
- [x] I have run `npm run test` to check if all tests are passing.
- [x] I have run `npx turbo build` to check if the website builds without errors.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Comment by AugustinMauroy** [comment]
Could you add blue sky logo on this story 
https://github.com/nodejs/nodejs.org/blob/main/apps/site/components/__design__/social-logos.stories.tsx

**Comment by TenzDelek** [comment]
> Could you add blue sky logo on this story https://github.com/nodejs/nodejs.org/blob/main/apps/site/components/__design__/social-logos.stories.tsx

done

**Review by AugustinMauroy — APPROVED** [review]
LGTM ! Thank for your contribution

File: apps/site/navigation.json [hunk 1]
```diff
@@ -77,6 +77,11 @@
       "icon": "linkedin",
       "link": "https://www.linkedin.com/company/node-js",
       "alt": "LinkedIn"
+    },
+    {
+      "icon": "bluesky",
+      "link": "https://bsky.app/profile/nodejs.org",
+      "alt": "Bluesky"
     }
```
Comment (avivkeller) [inline apps/site/navigation.json]: IMO Bluesky should go next to Twittter/X, as they are similar platforms (in terms of how they work)

Comment (TenzDelek) [reply apps/site/navigation.json]: image
done

**Review by avivkeller — APPROVED** [review]

**Review by shanpriyan — APPROVED** [review]
LGTM! :shipit:

**Review by bmuenzenmeyer — APPROVED** [review]

**Pull request merged**